ateam_ui/src/src/components/StatusComponent.vue

```diff
@@ -171,6 +171,7 @@ export default {
                     if (general)  errString = errString + "G";
                     if (hall)  errString = errString + "H";
                     if (encoder)  errString = errString + "E";
+                    errString = String(i);
```

Comment:
Is this line intentional?
It seems like the error info will be stripped and replaced with the current index

Reply:
Moved the motor index to the start of the error character sequence and resolved this in the process.